src/server.c

```diff
@@ -349,10 +349,20 @@ int dictSdsCompareKV(dictCmpCache *cache, const void *sdsLookup, const void *kv)
     return memcmp(sdsLookup, key2, l1) == 0;
 }
 
+static void trackDeallocationKV(dict *d, kvobj *kv) {
```

Comment:
it is rather short func, called only by other short func. i think we should merge them

Reply:
Done.